zb-db/src/main/java/io/zeebe/db/impl/rocksdb/transaction/TransactionalColumnFamily.java

```diff
@@ -148,6 +149,18 @@ public boolean exists(DbContext context, KeyType key) {
     return transactionDb.exists(handle, context, key);
   }
 
+  @Override
+  public boolean existsPrefix(DbKey keyPrefix) {
+    setPrefixExists(false);
+    whileEqualPrefix(
+        keyPrefix,
+        (key, value) -> {
+          setPrefixExists(true);
+          return false;
+        });
+    return prefixExists;
```

Comment:
Maybe, we should move this logic into `ZeebeTransactionDb` nex to the other database accesses.

Reply:
Agree :+1: Moved and added some tests.